test/phone_number_test.dart

```diff
@@ -107,6 +107,29 @@ void main() {
         expect(PhoneParser.fromRaw('33').isoCode, equals(IsoCode.FR));
         expect(PhoneParser.fromRaw('33').nsn, equals(''));
       });
+
+      test('should parse international numbers in a national or international format if given extra metadata for caller country',(){
+        expect(PhoneNumber.fromRaw('011 33 655 5705 76', callerCountry: IsoCode.US).international,
+            equals('+33655570576'));
+        expect(PhoneNumber.fromRaw('+33 655 5705 76', callerCountry: IsoCode.DE).international,
```

Comment:
there needs to be the same test with a german phone number (for the comment above)

Reply:
Added a few more tests with german numbers in different variations.